### PR TITLE
FIX: Incomplete datalad push information

### LINKED ISSUE #228: Eye-tracker data of pilot session 20 not properly uploaded
I'm getting

```
$ datalad get */*.tsv.gz
get(error): sub-001/ses-pilot020/func/sub-001_ses-pilot020_task-qct_recording-eyetracking_physio.tsv.gz (file) [not available; (Note that these git remotes have annex-ignore set: github)]
get(error): sub-001/ses-pilot020/func/sub-001_ses-pilot020_task-bht_recording-eyetracking_physio.tsv.gz (file) [not available; (Note that these git remotes have annex-ignore set: github)]
get(error): sub-001/ses-pilot020/dwi/sub-001_ses-pilot020_acq-highres_run-2__recording-eyetracking_physio.tsv.gz (file) [not available; (Note that these git remotes have annex-ignore set: github)]
action summary:
  get (error: 3)
```

### PR DESCRIPTION
**Pull request by oesteban**
These changes substantially improve the section about adding more data into the DataLad dataset.

In addition to that, it revises a few related aspects about physio and eyetracking.

Resolves: #228.
Relates: #235.
Relates: #185.

### CODE REVIEW

**Comment by oesteban** [comment]
I failed to mention #74. This PR opens a new section in the SOPs because we had a section for physio and ET, but it only covered physio.

I have added a section for ET, but it must be filled in with the procedure (I see that is part of #74).

File: docs/data-management/post-session.md [hunk 1]
```diff
@@ -163,7 +185,30 @@ As new sessions are collected, the corresponding BIDS structures MUST be saved w
...
+
+        Please read [*DataLad*'s `save` documentation](http://docs.datalad.org/en/stable/generated/man/datalad-save.html)
+
 - [ ] Push the new data to the remote storage (if your git containing *DataLad* and the Git annex is different from `origin`, e.g., `github`, replace the name below):
     ``` shell
+    datalad push --to ria-storage
     datalad push --to origin
     ```
+
+    !!! danger "Always double-check that data in the annex are uploaded to the RIA store"
```
Comment (esavary) [inline docs/data-management/post-session.md:214]: I think it would be useful to add the command below.

**Review by esavary — APPROVED** [review]

**Comment by oesteban** [comment]
> > I'm currently hitting [mmatl/pyrender#13](https://github.com/mmatl/pyrender/issues/13) and I believe this [psychopy/psychopy#5407](https://github.com/psychopy/psychopy/issues/5407) could be one cause.
> 
> I'm not sure I understood your last comment. Is it possibly related to another pull request?

totally, sorry about that.

**Comment by oesteban** [comment]
@esavary I've deleted the comments following mine, which you correctly identified as completely unrelated to the conversation. I've deleted them to reduce the noise on the mentioned repositories.

File: docs/data-management/post-session.md (hunk 1 above)
Comment (oesteban) [reply docs/data-management/post-session.md:214]: what do you mean?

Comment (esavary) [reply docs/data-management/post-session.md:214]: Is there a method to double check?

Comment (oesteban) [reply docs/data-management/post-session.md:214]: AFAIK you can clone the dataset somewhere else and test it pulls down the data :)

Also, when doing the push to ria-storage explicitly it should tell you if it failed.

**Pull request merged**